src/test/deploy/functions/validate.spec.ts

```diff
@@ -0,0 +1,137 @@
+import { expect } from "chai";
+import * as fsutils from "../../../fsutils";
+import * as validate from "../../../deploy/functions/validate";
+import * as projectPath from "../../../projectPath";
+import * as FirebaseError from "../../../error";
+import * as sinon from "sinon";
+
+// have to require this because no @types/cjson available
+// tslint:disable-next-line
+const cjson = require("cjson");
```

Comment:
Can this be 'import cjson = require("cjson")'? Helps us inch slightly closer to modern import/export syntax

Reply:
Good idea - unfortunately when I tried this it errored out because it doesn't have types so seems like the `import` part in a .ts file will look for types.